### PR TITLE
chore: migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: .github/workflows/docsify-cli.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: docsify-cli
+
+on: [ push, pull_request ]
```
Comment (jamesgeorge007) [inline .github/workflows/docsify-cli.yml:3]: Let's have the [workflow_dispatch](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch) event as well.

File: .github/workflows/docsify-cli.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  build:
+    runs-on: "${{ matrix.os }}"
+    strategy:
+      matrix:
+        os: [ macos-latest, ubuntu-latest ]
```
Comment (jamesgeorge007) [inline .github/workflows/docsify-cli.yml:10]:
```suggestion
        os: [ macos-latest, ubuntu-latest, windows-latest ]
```

**Review by jamesgeorge007 — CHANGES_REQUESTED** [review]
Looks good; left a couple of comments.

File: .github/workflows/docsify-cli.yml (hunk 1 above)
Comment (sy-records) [reply .github/workflows/docsify-cli.yml:3]: What needs is this used to fulfill?

Comment (jamesgeorge007) [reply .github/workflows/docsify-cli.yml]: It allows us to trigger workflows manually.

**Review by jamesgeorge007 — APPROVED** [review]

**Pull request merged**